### DISCUSSION TITLE
IDs de los pingüinos

### DISCUSSION THREAD

**Discussion by carlosmartellus** [IIC2133-PUC/2024-2#75] [Tarea 1] [answered]
Hola, el ingreso de los pinngüinos es por orden de ID o nuevamente es desordenado? Me refiero a happy

**Comment by CarlangaUC** [comment, marked as answer]
Hola!

Si te refieres al happy de cada pinguino asume que no vienen en orden necesariamente.

Saludos.

**Reply by carlosmartellus** [reply]
Hola, no me refiero a la felicidad de cada pingüino, me refiero si al momento de entrar, los pingüinos entrarán de forma ordenada en cuanto al ID, pero esto en el archivo happy

**Reply by CarlangaUC** [reply]
Entiendo, en la parte 2 cuando se habla de cuando entra un pinguino estos **entraran en orden**, si quieres corroborar esto fijate en los diversos test que se subieron para que veas mas o menos como son los inputs.

Saludos.
